packages/ffe-grid/less/ffe-grid.less

```diff
@@ -304,6 +304,14 @@
     .ffe-grid--condensed &--no-bottom-padding {
         padding-bottom: 0;
     }
+
+    &--top-padding {
+        padding-top: @ffe-grid-gutter-sm;
+
+        @media only screen and (min-width: @breakpoint-lg) {
+            padding-top: @ffe-grid-gutter-lg;
+        }
+    }
```

Comment:
This prop should probably use the `@ffe-condensed-grid-gutter-[sm|lg]` variable when the grid is condensed.